src/Jackett.Common/Indexers/TorrentSyndikat.cs

```diff
@@ -97,155 +93,178 @@ public TorrentSyndikat(IIndexerConfigurationService configService, WebClient w,
+
+                    var id = row.Value<string>("id");
+                    var comments = new Uri(SiteLink + "details.php?id=" + id);
+                    var seeders = row.Value<int>("seeders");
 
-                    var catStr = row.Children[0].FirstElementChild.GetAttribute("href").Split('=')[1].Split('&')[0];
-
-                    var qLink = row.Children[2].FirstElementChild;
-
-                    var descCol = row.Children[1];
-                    var torrentTag = descCol.QuerySelectorAll("span.torrent-tag");
-                    //Empty list gives string.Empty in string.Join
-                    var description = string.Join(", ", torrentTag.Select(x => x.InnerHtml));
-                    var qCommentLink = descCol.QuerySelector("a[href*=\"details.php\"]");
-                    var comments = new Uri(SiteLink + qCommentLink.GetAttribute("href").Replace("&hit=1", ""));
-
-                    var torrentDetails = descCol.QuerySelector(".torrent_details");
-                    var rawDateStr = torrentDetails.ChildNodes[1].TextContent;
-                    var dateStr = rawDateStr.Replace("von", "")
-                                            .Replace("Heute", "Today")
-                                            .Replace("Gestern", "Yesterday");
-                    var dateGerman = DateTimeUtil.FromUnknown(dateStr);
-                    var pubDateUtc = TimeZoneInfo.ConvertTimeToUtc(dateGerman, germanyTz);
-                    var longFromString = ParseUtil.GetLongFromString(descCol.QuerySelector("a[href*=\"&searchin=imdb\"]")?.GetAttribute("href"));
-                    var sizeFileCountRowChilds = row.Children[5].Children;
-                    var seeders = ParseUtil.CoerceInt(row.Children[7].TextContent);
-                    var link = new Uri(SiteLink + qLink.GetAttribute("href"));
-                    var files = ParseUtil.CoerceInt(sizeFileCountRowChilds[2].TextContent);
-                    var leechers = ParseUtil.CoerceInt(row.Children[8].TextContent);
-                    var grabs = ParseUtil.CoerceInt(row.QuerySelector("td:nth-child(7)").TextContent);
-                    var downloadVolumeFactor = globalFreeleech || row.QuerySelector("span.torrent-tag-free") != null
-                        ? 0 : 1;
-                    var size = ReleaseInfo.GetBytes(sizeFileCountRowChilds[0].TextContent);
                     var release = new ReleaseInfo
                     {
                         MinimumRatio = 1,
-                        MinimumSeedTime = 345600, //8 days
-                        Category = MapTrackerCatToNewznab(catStr),
-                        Link = link,
-                        Description = description,
-                        Title = qCommentLink.GetAttribute("title"),
-                        Comments = comments,
+                        MinimumSeedTime = 96 * 60 * 60,
+                        DownloadVolumeFactor = 1,
```

Comment:
In the old code I can see this site has global freeleech and per-torrent freeleech. Could you check it?

Reply:
Per-Torrent freeleech won't happen in the future. For global freeleech there is no API endpoint and TS won't include one for 2 days of freeleech per year.

Reply:
Ok